extension/src/popup/components/sendPayment/SendConfirm/TransactionDetails/index.tsx

```diff
@@ -71,6 +73,12 @@ const TwoAssetCard = ({
   const sourceAsset = getAssetFromCanonical(sourceCanon);
   const destAsset = getAssetFromCanonical(destCanon);
 
+  const isSourceAssetScam = useIsOwnedScamAsset(
```

Comment:
not related to this bug, but just noticed the warning icon wasn't showing in the confirmation screen for path payments